chroma-agent/chroma-agent.service

```diff
@@ -9,5 +9,6 @@ WantedBy=multi-user.target
 Type=simple
 ExecStart=/usr/bin/chroma-agent-daemon
 Restart=on-failure
+SyslogFacility=daemon
```

Comment:
Will this actually make any difference?  It already defaults to `daemon`.

Reply:
yeah... looks like `KeyError` it's still there. 

Wonder if I can get this silenced somehow:

```
Feb 27 12:18:02 localhost chroma-agent-daemon: /usr/lib/python2.7/site-packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
Feb 27 12:18:02 localhost chroma-agent-daemon: InsecureRequestWarning)
```

Reply:
Yes PLEASE!

Reply:
I think I found a way, working on it now.

Reply:
Tested locally, looks like latest patch does the trick.